File: maui-sample/Models/PortInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MauiSample.Models
{
  internal class PortInfo
  {
    public static int LocationPort { get; set; } = 9635;
    public static int LocationSecurePort { get; set; } = 9636;
    public static int ApiPort { get; set; } = 9637;
    public static int ApiSecurePort { get; set; } = 9638;
    public static int LocationV2Port { get; set; } = 9639;
    public static int LocationV2SecurePort { get; set; } = 9640;
  }
}

File: ios-native-sample/ios-native-sample/Models/PortInfo.swift
import Foundation

enum PortInfo: Sendable {
  nonisolated(unsafe) static var locationPort = 9635
  nonisolated(unsafe) static var locationSecurePort = 9636
  nonisolated(unsafe) static var apiPort = 9637
  nonisolated(unsafe) static var apiSecurePort = 9638
  nonisolated(unsafe) static var locationV2Port = 9639
  nonisolated(unsafe) static var locationV2SecurePort = 9640

  static func apply(registration: RegistrationDetails) {
    locationPort = registration.locationPort
    locationSecurePort = registration.locationSecurePort
    apiPort = registration.apiPort
    apiSecurePort = registration.apiSecurePort
    locationV2Port = registration.locationV2Port
    locationV2SecurePort = registration.locationV2SecurePort
  }
}
